src/migrate.ts

```diff
@@ -46,5 +46,5 @@ export async function migrate (args: string[]) {
       console.log(`[chel] Migrating... ${Math.round(numVisitedKeys / (numKeys / 10))}0% done`)
     }
   }
-  numKeys && console.log(`[chel] ${colors.green('Migrated:')} ${numKeys} entries`)
+  if (numKeys > 0) console.log(`[chel] ${(colors.green as (text: string) => string)('Migrated:')} ${numKeys} entries`)
```

Comment:
This `(colors.green as (text: string) => string` thing is insanely ugly, can you get rid of it somehow?